core/upgrade/elementor-3-re-migrate-globals.php

```diff
@@ -71,6 +71,11 @@ private function run() {
 		$meta_key = SettingsPageManager::META_KEY;
 		$kit_raw_settings = $kit->get_meta( $meta_key );
 
+		// Apply a default value for $kit_raw_settings in case it's not defined.
+		if ( empty( $kit_raw_settings )) {
+			$kit_raw_settings = array();
```

Comment:
```suggestion
			$kit_raw_settings = [];
```

Reply:
@matipojo unrelated, but linked to this file

$active_kit_id  is undefined
Suggest adding in the run function? unless there is a reason for this

$active_kit_id = Plugin::$instance->kits_manager->get_active_id();

Thanks

Reply:
How can it be `undefined`? the get_active_kit will create it if it does not exist.

Reply:
`$kit` is defined using `$kit = Plugin::$instance->kits_manager->get_active_kit();`, but `$active_kit_id` is not defined, but used here https://github.com/elementor/elementor/blob/3098d347d7af2db95299d36cc445165e9dcc1a6c/core/upgrade/elementor-3-re-migrate-globals.php#L102

Reply:
You are right, we will fix it. (@Aviad-Herman FYI)

Reply:
@matipojo looking at this [commit](https://github.com/elementor/elementor/commit/d7eeee2eadb5ba9819bc1f04239e7c013171e208)

The fix we spoke of above was in the code at some point, perhaps this got lost along the way?

Reply:
Made the required changes.